pysoa/client/client.py

```diff
@@ -297,8 +301,12 @@ def call_actions(self, service_name, actions, expansions=None, raise_action_erro
     def _perform_expansion(self, skeleton_response, expansions):
         # Perform expansions
         if expansions and hasattr(self, 'expansion_converter'):
-            objs_to_expand = self._extract_candidate_objects(skeleton_response.actions, expansions)
-            self._expand_objects(objs_to_expand)
+            try:
+                objs_to_expand = self._extract_candidate_objects(skeleton_response.actions, expansions)
+            except KeyError as keyError:
```

Comment:
ugh... camelCase? :-P
Just call it `error`

Reply:
thank you :)